compiler/dyno/include/chpl/types/ClassType.h

```diff
@@ -100,7 +100,7 @@ class ClassType final : public Type {
     return basicType_;
   }
 
-  /** Returns the verison this ClassType with the passed decorator */
+  /** Returns the version of this ClassType with the passed decorator */
```

Comment:
@mppf Was this what was meant here?

(Also, I don't know who to ping for review of this PR as a whole.)

Reply:
Yes that change is good